data/language/en-GB.yml

```diff
@@ -2280,3 +2280,6 @@ strings:
   2225: "Construction: Remove at current position"
   2226: "Construction: Select position"
   2227: "Trains reverse at signals"
+  2228: "Start Server"
```

Comment:
these stringid numbers will need to be updated since a few new ones have been added recently